src/BackupMetadata.cs

```diff
@@ -82,23 +82,39 @@ public class BackupMetadata : ICloneable
     /// </summary>
     public BackupFileTools.BackupType BackupType { get; set; }
 
+    internal BackupMetadata() { }
+
+    protected BackupMetadata(BackupMetadata backup)
+    {
+       CheckpointLsn = backup.CheckpointLsn;
+       DatabaseBackupLsn = backup.DatabaseBackupLsn;
+       DatabaseName = backup.DatabaseName;
+       FirstLsn = backup.FirstLsn;
+       LastLsn = backup.LastLsn;
+       PhysicalDeviceName = backup.PhysicalDeviceName;
+       ServerName = backup.ServerName;
+       StartTime = backup.StartTime;
+       BackupType = backup.BackupType;
+    }
+
     // used during testing
     public object Clone()
     {
       return MemberwiseClone();
     }
   }
 
-  public class StripedBackupSet
+  public class StripedBackupSet : BackupMetadata
   {
     public IEnumerable<BackupMetadata> StripedBackups { get; private set; }
 
-    private StripedBackupSet(IEnumerable<BackupMetadata> stripedBackups)
+    private StripedBackupSet(IEnumerable<BackupMetadata> stripedBackups) : base(stripedBackups.First())
     {
       StripedBackups = stripedBackups;
+      PhysicalDeviceName = null;

```

Comment:
Some discussion is needed here:
The only thing that differs between each "stripe" is the `PhysicalDeviceName`.  Why are you setting it to null here?  That is a confusing usage of the inherited value.  I think you can do better by adding some class hierarchy to this:

```C#
public abstract class BackupMetadata
{ /*all of the fields except PhysicalDeviceName*/}

public class SingleBackupMetadata : BackupMetadata
{ PhysicalDeviceName = backup.PhysicalDeviceName; }

public class StripedBackup : BackupMetadata
{  StripedBackups = stripedBackups; }
```